clients/fides-js/src/lib/events.ts

```diff
@@ -81,3 +81,21 @@ export const dispatchFidesEvent = (
     window.dispatchEvent(event);
   }
 };
+
+/**
+ * An alternative way to subscribe to Fides events. The same events are supported, except the callback
+ * receives the event details directly. This is useful in restricted environments where you can't
+ * directly access `window.addEventListener`.
+ *
+ * Returns an unsubscribe function that can be called to remove the event listener.
```

Comment:
can we add a usage example here, e.g `Fides.onFidesEvent("FidesUpdated", (data) => console.log(data))`

 Were you able to get this added in the `docs/fides.ts` file, too?

Reply:
Usage example with docs added 👍